openapi_tester/case_testers.py

```diff
@@ -1,97 +1,24 @@
 import logging
-import re
+from typing import Callable, Optional
 
 from openapi_tester.exceptions import CaseError
+from inflection import camelize, underscore, humanize, dasherize
 
 logger = logging.getLogger('openapi_tester')
 
 
-def is_camel_case(key: str, origin: str) -> None:
-    """
-    Asserts that a value is camelCased.
+def _create_tester(casing: str, handler: Callable[[str, Optional[bool]], str]) -> Callable[[str, str], None]:
+    """ factory function for creating testers """
+    def tester(key: str, origin: str) -> None:
+        logger.debug(f'Verifying that {origin} key `{key}` is properly {humanize(casing)}')
```

Comment:
```suggestion
        logger.debug('Verifying that %s key `%s` is properly %s', origin, key, humanize(casing))
```

I generally try to use the old style formatting for logger statements. Pretty much just because of [this](https://blog.pilosus.org/posts/2020/01/24/python-f-strings-in-logging/) blog post.

Reply:
TBH - pytest is so fast, this is a micro optimization. I would suggest targeting readers rather than machines in test code as a best practice. For me template strings are great.

Reply:
Yeah, the Sentry section is the most important IMO, but I guess this code won't realistically ever create a Sentry issue, so I agree.